go.mod

```diff
@@ -20,11 +20,11 @@ require (
 	github.com/fluxcd/kustomize-controller/api v1.4.0
 	github.com/fluxcd/pkg/apis/acl v0.5.0
 	github.com/fluxcd/pkg/apis/event v0.15.0
-	github.com/fluxcd/pkg/apis/kustomize v1.8.0
-	github.com/fluxcd/pkg/apis/meta v1.9.0
+	github.com/fluxcd/pkg/apis/kustomize v1.8.1-0.20250123112748-c55030369b58
+	github.com/fluxcd/pkg/apis/meta v1.9.1-0.20250127154341-0b0eba6810f4
 	github.com/fluxcd/pkg/http/fetch v0.14.0
 	github.com/fluxcd/pkg/kustomize v1.15.0
-	github.com/fluxcd/pkg/runtime v0.52.0
+	github.com/fluxcd/pkg/runtime v0.52.1-0.20250127144843-68389f802c21
```

Comment:
Need to import the proper tag after https://github.com/fluxcd/pkg/pull/858 is merged.